README.md

```diff
@@ -52,17 +48,35 @@ SNAPSHOT=true DEV=true PLATFORMS=darwin mage package
 The above assumes you are running on OS X. Put the platform in you are running on. This speeds up packaging as it only builds it for your platform. As soon as this is completed (it might take a while for the first time) navigate to `build/distributions` and unpackage the `.tar.gz`. Navigate into the elastic-agent directory and start the Elastic Agent:
 
 ```
-sudo ./elastic-agent -v
+KIBANA_HOST=http://localhost:5601 KIBANA_USERNAME=elastic KIBANA_PASSWORD=changeme ELASTICSEARCH_HOST=http://localhost:9200 ELASTICSEARCH_USERNAME=elastic ELASTICSEARCH_PASSWORD=changeme KIBANA_FLEET_SETUP=1 FLEET_SERVER_ENABLE=1 sudo ./elastic-agent container
 ```
 
-Currently a second command has to be run in parallel to setup fleet-server. Take the enrollment token and policy id you copied from Kibana and replace it in the command below:
+This will start up Elastic Agent with fleet-server and directly enroll it. In addition Fleet is setup.
+
+## Running Elastic Agent with fleet-server in container
+
+If you want to run Elastic Agent and fleet-server in a container but built Kibana from source, you have to add the following to your `config/kibana.dev.yml`:
 
 ```
-sudo ./elastic-agent enroll --enrollment-token {enrollment-token} --fleet-server http://elastic:changeme@localhost:9200 --fleet-server-policy {fleet-server-policy-id}
+server.host: 0.0.0.0
 ```
 
-This will start up fleet-server and the command should complete. After this, navigate to Kibana and check if the Elastic Agent with fleet-server shows up.
+This makes sure, Kibana is accessible from the container. Start Kibana as before but for Elasticsearch, run the following command:
+
+```
+yarn es snapshot -E xpack.security.authc.api_key.enabled=true -E http.host=0.0.0.0
+```
+
+This makes sure also Elasticsearch is accessible to the container.
+
+Start the Elastic Agent with the following command:
+
+```
+docker run -e KIBANA_HOST=http://{YOUR-IP}:5601 -e KIBANA_USERNAME=elastic -e KIBANA_PASSWORD=changeme -e ELASTICSEARCH_HOST=http://{YOUR-IP}:9200 -e ELASTICSEARCH_USERNAME=elastic -e ELASTICSEARCH_PASSWORD=changeme -e KIBANA_FLEET_SETUP=1 -e FLEET_SERVER_ENABLE=1 -e FLEET_SERVER_INSECURE_HTTP=1 docker.elastic.co/beats/elastic-agent:8.0.0-SNAPSHOT
```

Comment:
@blakerouse Why do I need here `FLEET_SERVER_INSECURE_HTTP=1`? Otherwise I get the following error:

```
2021-03-10T09:50:45.732Z	INFO	application/enroll_cmd.go:530	Fleet Server - Starting
2021-03-10T09:50:47.683Z	WARN	application/enroll_cmd.go:334	Remote server is not ready to accept connections, will retry in a moment.
```

If I run it directly from my machine, it seems to work without it.

Reply:
You should not need the `FLEET_SERVER_INSECURE_HTTP`.  Seems that the TLS based on hostname might be the issue. I will look into it.